tests/mbt/model/ccv_model.qnt

```diff
@@ -359,50 +469,71 @@ module ccv_model {
                 // start consumer1
                 EndAndBeginBlockForProvider(1 * Second, Set("consumer1"), Set())
             })
             .then(
                 all {
                     // consumer1 was started
                     assert(currentState.providerState.consumerStatus.get("consumer1") == RUNNING),
-                    // a packet was sent to consumer1
-                    assert(currentState.providerState.outstandingPacketsToConsumer.get("consumer1").length() == 1),
+                    // but no packet was sent to consumer 1
+                    assert(currentState.providerState.outstandingPacketsToConsumer.get("consumer1").length() == 0),
                     // the validator set on the provider was entered into the history
-                    assert(currentState.providerState.chainState.votingPowerHistory == List(InitialValidatorSet.put("node1", 50), InitialValidatorSet)),
-                    // deliver the packet
-                    DeliverVscPacket("consumer1")
+                    assert(currentState.providerState.chainState.votingPowerHistory == List(InitialValidatorSet.put("node1", 150), InitialValidatorSet)),
+                    // change voting power on provider again
+                    VotingPowerChange("node1", 50).then(
+                        // end another block
+                        EndAndBeginBlockForProvider(1 * Second, Set(), Set())
+                    ).then(
+                        // deliver packet to consumer1
+                        DeliverVscPacket("consumer1")
+                    )
                 }
             )
             .then(
                 all {
                     // make sure the packet was removed from the provider
                     assert(currentState.providerState.outstandingPacketsToConsumer.get("consumer1").length() == 0),
                     // ensure the maturation time was entered on the consumer
-                    assert(currentState.consumerStates.get("consumer1").maturationTimes.keys().size() == 1),
+                    assert(currentState.consumerStates.get("consumer1").maturationTimes.length() == 1),
                     // the validator set was put as the current validator set
-                    assert(currentState.consumerStates.get("consumer1").chainState.currentValidatorSet == InitialValidatorSet.put("node1", 50)),
-                    // advance time on provider until the unbonding period is over
-                    EndAndBeginBlockForProvider(UnbondingPeriodPerChain.get("consumer1"), Set(), Set()),
+                    assert(currentState.consumerStates.get("consumer1").chainState.currentValidatorSet == InitialValidatorSet.put("node1", 200)),
+                    // advance time on provider and consumer until unbonding period is over - ensure that the consumer and provider
+                    // stay in sync relative to each other
+                    4.reps(
```

Comment:
What's the benefit of advancing `UnbondingPeriod / 4` 4 times instead of advancing `UnbondingPeriod` at once?

Reply:
Added comment, please resolve if that clears it up

Reply:
👍 Makes sense. Thanks!